command/agent/auth/auth.go

```diff
@@ -169,12 +201,14 @@ func (ah *AuthHandler) Run(ctx context.Context, am AuthMethod) error {
 			}
 		}
 
-		secret, err := clientToUse.Logical().Write(path, data)
-		// Check errors/sanity
-		if err != nil {
-			ah.logger.Error("error authenticating", "error", err, "backoff", backoff.Seconds())
-			backoffOrQuit(ctx, backoff)
-			continue
+		if secret.Auth == nil {
```

Comment:
Can we include a comment here on when this case gets hit?

Reply:
Done!